db/image_job.sql

```diff
@@ -0,0 +1,54 @@
+CREATE TABLE IF NOT EXISTS IMAGE_JOB(
+    job_id  BIGINT NOT NULL REFERENCES JOB(job_id),
+    /*job_status INTEGER NOT NULL DEFAULT 0,*/
+    CONSTRAINT pk_image_job PRIMARY KEY(job_id)
+);
+
+CREATE TABLE IF NOT EXISTS IMAGE_STATUS(
+    imstat_id    INTEGER CONSTRAINT pk_imstat PRIMARY KEY,
+    imstat_name  TEXT NOT NULL
```

Comment:
I think I would change that to `image_stat_id` and `image_stat_name`

Reply:
"imstat" may be too short a name, but giving it a name like "image_stat_id" would imply "Image: status id" rather than "Image Status: id".